test/cppsuite/test_harness/test.h

```diff
@@ -57,6 +57,9 @@ class test : public database_operation {
     public:
     test(const std::string &config, const std::string &name)
     {
+        /* Set the program name for error messages. */
+        progname = name.c_str();
```

Comment:
I'm wondering whether the destructor should set `progname` back to its default value. Otherwise, if we do a `testutil_assert` call after the test, we'll read into freed memory.

The other alternative is to follow the convention of the other tests by setting `progname` to the actual program name by calling `testutil_set_progname` (in this case, it would be called `run`).

Reply:
Happy to follow the convention of the other tests.

Reply:
e1a708f